docs/index.md

```diff
@@ -4,6 +4,8 @@ A toolkit of **great libraries** to start building **Typelevel** apps on JVM, No
 
 Our very own flavour of the [Scala Toolkit].
 
+**IMPORTANT**: The toolkit does not yet support Scala Native 0.5.x series.
```

Comment:
We can use a Laika directive here.

https://typelevel.org/Laika/latest/07-reference/01-standard-directives.html#callout

Would be cool if Laika supports the GitHub style ones too (maybe it already does or there is feature request, not sure).